crates/oxc_ast/src/ast/js.rs

```diff
@@ -1069,6 +1070,16 @@ pub struct VariableDeclarator<'a> {
     pub definite: bool,
 }
 
+#[derive(Debug, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize), serde(tag = "type", rename_all = "camelCase"))]
+pub struct UsingDeclaration<'a> {
```

Comment:
Let's add a doc comment with a link to the spec for future readers.

Reply:
More specifically, add `https://github.com/tc39/proposal-explicit-resource-management` here for future reference.

Reply:
done